File: Python/Maths/Area of Triangle - (All sides known or Herons Formula).py
import math
a = float(input("Enter the value of a: "))
b = float(input("Enter the value of b: "))
c = float(input("Enter the value of c: "))

if (a+b) > c and (b+c) > a and (c+a) > b:
    s = (a+b+c)/2
    area = math.sqrt(s * (s-a) * (s-b) * (s-c))
    print("The area of triangle is =", round(area, 2))
else:
    print("Invalid Triangle")

File: Java/Maths/Area of Triangle - (All sides known or Herons Formula).java
import java.util.Scanner;
class triangleHF {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the value of a: ");
        double a = input.nextDouble();
        System.out.print("Enter the value of b: ");
        double b = input.nextDouble();
        System.out.print("Enter the value of c: ");
        double c = input.nextDouble();

        if( (a+b) > c && (b+c) > a && (c+a) > b ){
            double s = (a+b+c)/2;
            double area = Math.sqrt(s * (s-a) * (s-b) * (s-c));
            System.out.println("The area of triangle is = " + area);
        } else {
            System.out.println("Invalid Triangle");
        }
    }
}